### PR TITLE
replaced kubernetes/pkg/util/strings with kubernetes/util/strings

### LINKED ISSUE #73385: Replace kubernetes/pkg/util/strings with k8s.io/utils/tree/master/strings
At some point, the code in https://github.com/kubernetes/kubernetes/tree/master/pkg/util/strings/ was moved to https://github.com/kubernetes/utils/tree/master/strings

However we have not started using the code in the kubernetes/utils library. We should switch over to the vendored library and delete the duplicate code in the main repo.

### PR DESCRIPTION
**Pull request by chaopli**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
replaced kubernetes/pkg/util/strings with kubernetes/util/strings as mentioned https://github.com/kubernetes/kubernetes/issues/73385
**Which issue(s) this PR fixes**:

Fixes #73385 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/strings/BUILD [hunk 1]
```diff
@@ -13,7 +13,7 @@ go_library(
         "line_delimiter.go",
         "strings.go",
     ],
-    importpath = "k8s.io/kubernetes/pkg/util/strings",
+    importpath = "k8s.io/kubernetes/util/strings",
```
Comment (BenTheElder) [inline pkg/util/strings/BUILD]: pretty sure this change is wrong.

Comment (BenTheElder) [reply pkg/util/strings/BUILD]: we should be removing this package instead.

Comment (dims) [reply pkg/util/strings/BUILD]: right. this should be `importpath = "k8s.io/util/strings",`

Comment (BenTheElder) [reply pkg/util/strings/BUILD]: Pretty sure package should be `"k8s.io/kubernetes/pkg/util/strings"` like it was, but if we want to switch everything to `k8s.io/utils/strings` we can remove this whole directory and just change the imports everywhere else.

Comment (dims) [reply pkg/util/strings/BUILD]: yes, that was the intention of the issue. looks like i was not clear enough.

Comment (chaopli) [reply pkg/util/strings/BUILD]: Thanks @dims . While I cannot find either the k8s.io/utils/strings package nor the k8s.io/kubernetes/pkg/util/strings. Am I missing anything here?

Comment (dims) [reply pkg/util/strings/BUILD]: github url : https://github.com/kubernetes/utils/tree/master/strings
package name : k8s.io/utils/strings

Comment (chaopli) [reply pkg/util/strings/BUILD]: Thanks. I see. So it's under another repo. And the original package was under this repo's pkg/util/strings, which has already gone, right?

Comment (dims) [reply pkg/util/strings/BUILD]: @chaopli yes pull from new repo. looks like your CLA is not setup right as well. please look at tips and contact here to resolve it:
https://github.com/kubernetes/community/blob/master/CLA.md#troubleshooting

**Comment by dims** [comment]
looks like this was taken care of in https://github.com/kubernetes/kubernetes/pull/73455

/close

**Pull request closed without merging**